frontend/src/pages/legacy/en-US.md

```diff
@@ -0,0 +1,17 @@
+## Legacy
+
+Hello there! Once upon a time, you installed an add-on called Test Pilot for Firefox.
+
+Well the Test Pilot program for trying experimental features in Firefox is back, and we thought you might like to check it out.
+The old add-on has been uninstalled for you. If you&apos;d like the new one, just click the link.
```

Comment:
Please link this to https://testpilot.firefox.com.

Reply:
You want me to delete the like 3-6 and replace it with the link?

Reply:
Just replace the word "link" with [link](https://testpilot.firefox.com)